lib/rex/ui/interactive.rb

```diff
@@ -128,6 +128,12 @@ def detach
   attr_accessor :on_print_proc
   attr_accessor :on_command_proc
 
+  #
+  # A function to be run when the session hits an error
+  #
+
```

Comment:
```suggestion
```

There shouldn't be any whitespace between the documentation and the accessor

This is missing the type metadata here too :+1:

```suggestion
  # @return [Proc,nil] A function to be run when the session hits an error
  attr_accessor :on_error_proc

```

Docs on usage - https://github.com/lsegal/yard/blob/2d197a381c5d4cc5c55b2c60fff992b31c986361/docs/GettingStarted.md#documenting-attributes